README.md

```diff
@@ -14,6 +14,7 @@ These protocols define various components of the [vac](https://vac.dev) stack.
  - [remote log](./remote-log.md) - Remote replication of local logs.
  - [mdf](./mdf.md) - Metadata field for [MVDS](./mvds.md) messages. 
  - [waku](./waku.md) - ÐΞVp2p wire protocol, substituting [EIP-627](https://eips.ethereum.org/EIPS/eip-627).
+ - [edf](./edf.md) - Envelope data field specification for [waku](./waku.md).
```

Comment:
`edf` seems a bit too...precise and concise to me. Can we make this more descriptive for its purpose?

Reply:
@oskarth any suggestions? once this and mailserver is merged I was going to group them together under a waku folder probably.

Reply:
Not right now, I think it depends on what we see the purpose of this spec being.

`wedf` is a placeholder I suppose but eh